content/guides/development.md

```diff
@@ -240,7 +240,149 @@ T> Now that your server is working, you might want to give [Hot Module Replaceme
 
 ### Using webpack-dev-middleware
 
-?> Familiar with `webpack-dev-middleware`? We need your help! Please submit a PR to fill in the missing instructions and example here. Make sure to keep it simple as this guide is intended for beginners.
+`webpack-dev-middleware` is a wrapper that will emit files processed by webpack to a server. This is used in `webpack-dev-server` internally, however it's available as a separate package to allow more custom setups if desired. We'll take a look at an example that combines webpack-dev-middleware with an express server.
+
+Let's install `express` and `webpack-dev-middleware` so we can get started:
+
+``` bash
+npm install --save-dev express webpack-dev-middleware
+```
+
+Now we need to make some adjustments to our webpack configuration file in order to make sure the middleware will function correctly:
+
+__webpack.config.js__
+
+``` diff
+  const path = require('path');
+  const HtmlWebpackPlugin = require('html-webpack-plugin');
+  const CleanWebpackPlugin = require('clean-webpack-plugin');
+
+  module.exports = {
+    entry: {
+      app: './src/index.js',
+      print: './src/print.js'
+    },
+    devtool: 'inline-source-map',
+    plugins: [
+      new CleanWebpackPlugin(['dist']),
+      new HtmlWebpackPlugin({
+        title: 'Output Management'
+      }),
+      new webpack.HotModuleReplacementPlugin()
+    ],
+    output: {
+      filename: '[name].bundle.js',
+      path: path.resolve(__dirname, 'dist'),
++     publicPath: 'http://localhost/'
+    }
+  };
+```
+
+This will tell the middleware to serve our files on `http://localhost:3000` (the port number we'll specify later), but before we can actually see the files, we need to setup `express` and make sure it's running.
+
+__project__
+
+``` bash
+  webpack-demo
+  |- package.json
+  |- webpack.config.js
++ |- server.js
+  |- /dist
+  |- /src
+    |- index.js
+    |- print.js
+  |- /node_modules
+```
+
+__server.js__
+
+``` js
+const express = require('express');
+const webpack = require('webpack');
+const webpackDevMiddleware = require('webpack-dev-middleware');
+
+const app = express();
+const config = require('../webpack.config.js');
+const compiler = webpack(config);
+
+// Tell express to use the webpack-dev-middleware and use the webpack.config.js
+// configuration file as a base.
+app.use(webpackDevMiddleware(compiler, {
+  publicPath: config.output.publicPath
+}));
+
+// Serve the files on port 3000.
+app.listen(3000, function () {
+  console.log('Example app listening on port 3000!');
```

Comment:
Maybe throw a line break at the end of the statement so there's a little space between your and webpack's, e.g.:

``` js
console.log('Example app listening on port 3000!\n');
```

to generate below:

``` bash
Example app listening on port 3000!

webpack built 27b137af6d9d8668c373 in 1198ms
Hash: 27b137af6d9d8668c373
...
```

(somewhat nitpicky, I know -- feel free to ignore 😆 )